http/server.go

```diff
@@ -0,0 +1,80 @@
+package http
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/drand/drand/protobuf/drand"
+)
+
+// New creates an HTTP handler for the public Drand API
+func New(ctx context.Context, client drand.PublicClient) (http.Handler, error) {
+	pkt, err := client.Group(ctx, &drand.GroupRequest{})
+	if err != nil {
+		return nil, err
+	}
+	if pkt == nil {
+		return nil, fmt.Errorf("Failed to retrieve valid GroupPacket")
+	}
+
+	handler := handler{ctx, client, pkt}
+
+	mux := http.NewServeMux()
+	//TODO: aggregated bulk round responses.
+	mux.HandleFunc("/PublicRand", handler.PublicRand)
+	mux.HandleFunc("/Group", handler.Group)
+	return mux, nil
+}
+
+type handler struct {
+	ctx       context.Context
+	client    drand.PublicClient
+	groupInfo *drand.GroupPacket
+}
+
+func (h *handler) PublicRand(w http.ResponseWriter, r *http.Request) {
+	// Get the round.
+	round, ok := r.URL.Query()["round"]
+	if !ok || len(round) == 0 {
```

Comment:
So right now the behavior of the http endpoint AND grpc endpoint is that if you don't specify the round number, then the drand node returns the latest one. Should we keep the same behavior ? or make a `/latest` ?

Reply:
On this topic, super important to make this in the form of:

`/PublicRand/<Round>`
`/PublicRand/latest`

This way we can actually serve fully static content. The API could just be flipped then to anything that serves a filesystem folder and drand just write to it. And I am thinking S3 with 0 config (and I assume other storage options out there are similar).

Reply:
Can we make it `/public/<round>` and `/public/latest` ? just nitpicking here :D

Reply:
yes :)